docs/CONTRIBUTING.md

```diff
@@ -1,21 +1,271 @@
-# Contributing
+# Contributing to the web-features project
 
 Thanks for your interest in contributing to this project!
 Before you contribute, consider the following:
 
-- Help us create a kind, welcoming, and productive project.
+* Help us create a kind, welcoming, and productive project.
+ 
   You can do this in part by following the [W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/), which governs conduct on this project.
 
-- All contributions to this project are licensed under the terms of the Apache License, Version 2.0.
+* All contributions to this project are licensed under the terms of the Apache License, Version 2.0.
+ 
   [Read `LICENSE.txt` for details](../LICENSE.txt).
 
-## Get started
+There are multiple ways to contribute to the web-features repository, such as:
 
-The project is in its early stages, so we’re still developing tools and processes to help improve and expand our work.
-To get started contributing, take a look the following:
+* Adding a missing feature to the repository, from scratch.
+* Adding a missing feature to the repository, by updating a feature that was already drafted.
+* Updating an existing feature, for example to change its name, description, support data, or other fields.
+* Reviewing a pull request that was submitted by someone else, to check if the feature is correctly authored.
 
-* Go to [feature guidelines](./README.md#feature-guidelines).
-* Go to [open issues](https://github.com/web-platform-dx/web-features/issues) to propose new feature groups or report a bug.
-* Go to [open PRs](https://github.com/web-platform-dx/web-features/pulls) to review incoming feature groups and updates.
+In any case, **thank you** for wanting to help. This document will guide you through the process of contributing to the web-features repository.
 
-Also consider joining [the WebDX Community Group](https://www.w3.org/community/webdx/).
+Also consider joining the [WebDX Community Group](https://www.w3.org/community/webdx/).
+
+## What makes a feature
```

Comment:
It could be worth completing the section to say that, at least for the time being, we're more directly focusing on existing features: those that have some level of support somewhere, and that are typically already somewhat documented on MDN (and/or Can I Use).

What triggered this comment is that the part on setting the status below suggest to look at compatibility tables at the end of MDN pages related to the new feature at hand.